### PR TITLE
travis: Remove s390x build

### PR DESCRIPTION
**Pull request by maflcko**
This has been discussed in the last meeting.

Refer to the commit body for more details.

### CODE REVIEW

**Comment by hebasto** [comment]
Do I understand correctly that the last commit fixes _manual_ s390x builds?

**Comment by MarcoFalke** [comment]
It fixes *qemu* builds. Manual and automatic

**Comment by decryp2kanon** [comment]
could you explain why we should remove this? @MarcoFalke

**Comment by MarcoFalke** [comment]
* Running it on travis has been unreliable (search for s390x issues in this repo)
* travis org is shutting down and there is no s390x on travis com
* the test is more meaningful if run either locally in qemu or on any other arch that has different endianness

**Comment by decryp2kanon** [comment]
Concept ACK

**Comment by decryp2kanon** [comment]
this file is removing too?
`ci/test/00_setup_env_s390x.sh`

File: .travis.yml [hunk 1]
```diff
@@ -72,13 +72,6 @@ jobs:
         FILE_ENV="./ci/test/00_setup_env_arm.sh"
         QEMU_USER_CMD=""
 
-    - stage: test
-      name: 'S390x  [GOAL: install]  [buster]  [unit tests, functional tests]'
-      arch: s390x  # Can disable QEMU_USER_CMD and run the tests natively without qemu
-      env: >-
-        FILE_ENV="./ci/test/00_setup_env_s390x.sh"
```
Comment (decryp2kanon) [inline .travis.yml:79]: this file is removing too?

Comment (MarcoFalke) [reply .travis.yml:79]: I'd like to keep it so that it can be executed easier, if needed

**Pull request merged**